front-end/src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
+// import logo from "./../assets/images/logo.svg";
```

Comment:
You can add `NODE_SRC` as a env variables. That will allow you to import from wherever you specify.

For example. NODE_SRC=/src allows you to reference from src, instead of the lengthy `../../../` etc